<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Prompt</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ข้อมูลครุภัณฑ์</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>รหัสครุภัณฑ์ :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="asset_code">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>ชื่อครุภัณฑ์ :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="name">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>รายละเอียด :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="detail">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>ห้อง :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="room">
                 <property name="font">
                  <font>
                   <family>Prompt</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>อาคาร :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="building">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>ละติจูด :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="latitude">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>ลองจิจูด :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="longitude">
               <property name="font">
                <font>
                 <family>Prompt</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>เลือก(รหัส) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="id">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_clear">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ล้าง</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btn_save">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>บันทึก</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_update">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>อัพเดท</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_delete">
             <property name="font">
              <font>
               <family>Prompt</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ลบ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="font">
            <font>
             <family>Prompt</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
